controllers/Articles/articlesController.js

```diff
@@ -24,6 +26,11 @@ class ArticlesController {
       const createdArticles = await articleModelManager.createArticle(articleDetails);
       generateTags(req.body.tags, createdArticles.dataValues.id);
       if (createdArticles) {
```

Comment:
Sorry for catching this now. Can we have the notify users function, as its own middleware? After a successful article creation, next is called and the newly created article is added to the request object.

Reply:
no problem rex, I have done this.